client/state/ui/drop-zone/reducer.js

```diff
@@ -17,10 +13,17 @@ import {
 
 import { createReducer } from 'state/utils';
 
-const isVisible = createReducer( false,
+// Can be improved with `keyedReducer` instead of state spread.
```

Comment:
~What's the status on this? :)~ Just read the note below. I'd prefix this comment with `TODO(biskobe)` though.